nxdrive/engine/processor.py

```diff
@@ -580,6 +581,17 @@ def _direct_transfer_end(
                 session = self.dao.update_session(doc_pair.session)
             elif cancelled_transfer:
                 session = self.dao.decrease_session_counts(doc_pair.session)
+            self.remote.metrics.send(
+                {
+                    "NX-metric-drive.request": json.dumps(
```

Comment:
To ease maintenance, you should move `NX-metric-drive.request` to a constant in `metrics.py`. Maybe `REQUEST` is a good name for that.
Same for `GLOBAL`.